airbyte-integrations/connectors/destination-snowflake/src/main/java/io/airbyte/integrations/destination/snowflake/SnowflakeInsertDestination.java

```diff
@@ -1,48 +0,0 @@
-/*
- * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
- */
-
-package io.airbyte.integrations.destination.snowflake;
```

Comment:
For context, through code inspect this class doesn't appear to be in use so removal makes sense